Test/TestConfigurationUtils.ps1

```diff
@@ -319,26 +319,47 @@ function Clear-TestConfiguration {
     Wait-RemoteInterfaceIP -Session $Session -AdapterName $SystemConfig.AdapterName
 }
 
+function Install-ComputeServices {
```

Comment:
I'm thinking that we shouldn't add anything to TestConfigurationUtils file. If anything, it should shrink. Maybe let's use one of the files in ComputeNode/ directory: https://github.com/Juniper/contrail-windows-ci/tree/master/Test/Utils/ComputeNode

Reply:
Ok, I'd move it to better location.